### ISSUE TITLE
Possible bug in Houston 2.0.3?

### ISSUE THREAD

**Issue by ffxsam** [gterrono/houston#297]
If I remove a collection by running `db.the_collection.drop()` in Mongo, and remove the declaration itself:

    TheCollection = new Mongo.Collection('the_collection');

Houston still shows the collection, even after a server restart.

**Comment by gterrono** [comment]
Yeah, good catch. I don't think we handle that. I don't know of a way to do it right when it's dropped, but we should probably check on restart at least.
